bftclient/include/bftclient/fake_comm.h

```diff
@@ -159,11 +161,13 @@ inline std::vector<uint8_t> createReply(const MsgFromClient& msg, std::vector<ui
 
 inline void immideateBehaviour(const MsgFromClient& msg, IReceiver* client_receiver) {
```

Comment:
Please fix a typo in 'immideate' (does not relate to your changes)

Reply:
done.